dev_environment/cfg/payload_traffic_mirroring.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "host_name": "<hostname>",
+        "iface": "<datapath interface name>",
+        "bpf_programs": {
+            "tc_ingress": [
+                {
+                    "name": "traffic_mirroring",
+                    "seq_id": 2,
+                    "artifact": "traffic_mirroring.tar.gz",
+                    "cmd_start": "mirroring",
+                    "version": "latest",
+                    "user_program_daemon": true,
+                    "admin_status": "enabled",
+                    "prog_type": "tc",
+                    "cfg_version": 1,
+                    "start_args": {
+                        "direction": "ingress",
+                        "map-name": "/sys/fs/bpf/tc/globals/tc_ingress_root_array",
```

Comment:
L3AFD has `BpfMapDefaultPath: "/sys/fs/bpf"` so why is the "/sys/fs/bpf" prefix needed here?